_pytest/doctest.py

```diff
@@ -291,6 +295,26 @@ def _get_allow_bytes_flag():
     return doctest.register_optionflag('ALLOW_BYTES')
 
 
+def _get_report_choices_keys():
+    """
+    Returns the report choices keys. Separate function (see just below) because this is used to declare options and
+    we can't import the doctest module at that time.
+    """
+    return ('udiff', 'cdiff', 'ndiff', 'only_first_failure', 'none', )
+
+def _get_report_choices():
+    """
+    See `_get_report_choices_keys` to understand why this strange implementation. Be careful with order, the values
+    order should be the same as the returned keys above.
+    """
+    import doctest
+    return dict(
+        zip(
+            _get_report_choices_keys(),
+            (doctest.REPORT_UDIFF, doctest.REPORT_CDIFF, doctest.REPORT_NDIFF, doctest.REPORT_ONLY_FIRST_FAILURE, 0, )
+        )
+    )
```

Comment:
I will try something else about this, hold on. Will opt for the roof also, will get fresher code :D